Makefile.osl

```diff
@@ -0,0 +1,322 @@
+# VERSION defines the project version for the bundle.
```

Comment:
There is a way to import another makefiles, this way we could import the other one and only update what is needed to build the product image, wdyt?

Reply:
Using 
`.INCLUDE : Makefile`
we can import the variables but it seems that it doesn't import the targets so we have to put something like 
`@$(MAKE) -f Makefile $@`
for every unchanged target.
Do yo know a better way?